canonical-websites/www.ubuntu.com
# Add animation to Ubuntu product month takeover

## Done

- Added animation to Ubuntu product month takeover

## QA

- Pull code and run `./run`
- Check that the takeover has a little animation
- Check that it works in [all browsers we support](https://github.com/canonical-webteam/practices/blob/master/coding/browser-support.md)
